plyer/platforms/win/libs/screenmetric.py

```diff
@@ -0,0 +1,10 @@
+__all__ = ('get_SystemMetrics')
```

Comment:
is it ok to create this file? or should I put this function in `win_api_defs.py` and then in `win/sysinfo.py` use `plyer.platforms.win.libs import win_api_defs` instead of `plyer.platforms.win.libs import screenmetric`.